src/index.js

```diff
@@ -8,30 +8,51 @@ import {runScripts} from './transformScriptTags';
 function processOptions(options) {
   // Parse preset names
   const presets = (options.presets || []).map(presetName => {
-    if (typeof presetName === 'string') {
-      var preset = availablePresets[presetName];
-      if (!preset) {
-        throw new Error(`Invalid preset specified in Babel options: "${presetName}"`);
+    let preset = null;
+
+    if (presetName instanceof Array && typeof presetName[0] === 'string') {
```

Comment:
Can you please use `Array.isArray` here? `instanceof Array` will fail across iframes

I'd suggest doing this:
```
const isArray = Array.isArray || arg => Object.prototype.toString.call(arg) === '[object Array]';
```

Then using `isArray(presetName)`